examples/tomo/niftyrec_reconstruct.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Example reconstruction with NiftyRec."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+import odl
+from odl.tomo.operators.spect_trafo import AttenuatedRayTransform
+
+
+# Create SPECT geometry and reconstruction domain
+det_nx_pix = 64
+det_ny_pix = 64
+det_nx_mm = 2
+det_radius = 200
+n_proj = 64
+det_param = det_nx_mm * det_nx_pix / 2
```

Comment:
Is there a reason for this way of doing the input? In the other examples we give the extent directly.